### ISSUE TITLE
Create new `DividerTitleCard` card type

### ISSUE THREAD

**Issue by CollinHeist** [CollinHeist/TitleCardMaker#326]
I don't yet know what to call this card type, but it should be fairly simple, and based on "title cards" featured in Overlord season 3 - for example:

<img width="600" src="https://user-images.githubusercontent.com/17693271/232343943-275fd5c4-38df-4c9a-b79c-d2e99d149d55.jpg">

Although white/red should not be the default color. A rough ImageMagick command to do this (haven't tested for more than 3 minutes):

```bash
convert in.jpg \
-background transparent \
-font "hiragino-mincho-w3.ttc" \
\( -gravity east \
-pointsize 100 \
-stroke black \
-strokewidth 8 \
label:"Season 1\nEpisode 1" \
-pointsize 225 \
-gravity center \
label:"|" \
-gravity west \
-pointsize 100\
label:"Example\nTitle" \
+smush 25 \
-bordercolor transparent \
-border 50x50 \
-blur 0x4 \) \
-gravity southeast \
-composite \
\( -gravity east \
-pointsize 100 \
-fill white \
-stroke none \
-strokewidth 8 \
label:"Season 1\nEpisode 1" \
-pointsize 225 \
-gravity north label:"|" \
-gravity west \
-pointsize 100 \
label:"Example\nTitle" \
+smush 31 \
-bordercolor transparent \
-border 50x50 \) \
-gravity southeast \
-composite out.jpg
```

With the under stroke being generated by a blurred version of the overlaying text. Theoretically the index/title text position could easily be swapped around on the image and relative to each other.

**Comment by CollinHeist** [comment]
I ended up having to modify the above command quite a bit, but here's an example of the finished card:

![temp](https://user-images.githubusercontent.com/17693271/232378485-a9a737dc-9faf-47c2-b639-7df3d3ffb194.jpg)

I also implemented three extras, `stroke_color`, `title_text_position` and `text_position`. Here's an example showcasing all three:
```yaml
extras:
  stroke_color: crimson # Defaults to black
  title_text_position: right # Defaults to left
  text_position: left # Defaults to lower right
```
![temp](https://user-images.githubusercontent.com/17693271/232379117-b9a8e551-07c8-4152-8f3b-9c3cd1509499.jpg)

With `title_text_position` only being `left` or `right`, and just controls which side of the divider the title text appears. `text_position` can be `upper left`, `upper right`, `right`, `lower right`, `lower left`, or `left` And controls where the combined text appears on the image.
